### ISSUE TITLE
Lidar sensor doesn't recognize map made by RoadRunner

### ISSUE THREAD

**Issue by hyeon-ji-shin** [carla-simulator/ros-bridge#719]
Hi I want to know how to sense the map made by RoadRunner, for carla-ros-bridge.

The problem is that my lidar sensor can't find parts of map(such as barricade).
(but rgb-d camera, ... works!)

rostopic echo /carla/ego_vehicle/lidar
result -> data: []

I add multiple cylinders in unreal engines for test, then lidar sensor find only cylinders.

How can I do to sense other things?

![Screenshot from 2024-01-17 13-57-16](https://github.com/carla-simulator/ros-bridge/assets/75267548/3349385b-7964-4d9b-981f-9f9c450bb3a8)
![Screenshot from 2024-01-17 13-59-40](https://github.com/carla-simulator/ros-bridge/assets/75267548/23821337-564d-435b-abe7-08e1a419b2b1)

**Comment by Billthekidz** [comment]
Although recommended, road runner is owned by matworks. 
The main CARLA issues may be a better place to ask

**Comment by hyeon-ji-shin** [comment]
> Although recommended, road runner is owned by matworks. The main CARLA issues may be a better place to ask

Thanks for your comments and attention! 😺 

Fortunately, I've solved this issue before.
(As you said, I think it was reasonable to ask to the main CARLA issues. 😅 )

I found that many ways of types can support to import maps on CARLA.
① .fbx ② .udatasmith ③or just using command ('make import')

**I used .fbx** file that day.
Re-creating a custom map by **.udatasmith** will work.

I hope this comment will help someone. :D
